sdk/monitor/Azure.Monitor.OpenTelemetry.Distro/Azure.Monitor.OpenTelemetry.Distro/Azure.Monitor.OpenTelemetry.Distro.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>An OpenTelemetry .NET distro that exports to Azure Monitor</Description>
+    <AssemblyTitle>AzureMonitor OpenTelemetry Distro</AssemblyTitle>
+    <Version>1.0.0-alpha.1</Version>
+    <PackageTags>Azure Monitor OpenTelemetry Exporter ApplicationInsights</PackageTags>
```

Comment:
Add distro here.

Reply:
Do we want both Exporter and Distro here or only Distro?